src/python/tools/Report_InvalidUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,18 @@
+url	relativeFilepath
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\CommonCpp.props
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\NativeBridge\NativeBridge.vcxproj
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\python\nimbusml.pyproj
+http://www.nvidia.com/Drivers	.\src\DotNetBridge\MessageValidator.cs
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearBinaryClassifier.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearClassifier.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearRegressor.txt
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearbinaryclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\internal\core\linear_model\fastlinearregressor.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearbinaryclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearclassifier.py
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\nimbusml\linear_model\fastlinearregressor.py
+http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure-	.\src\python\docs\docstrings\PcaAnomalyDetector.txt
```

Comment:
>http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure- [](start = 0, length = 68)

url is different in source file

Reply:
Indeed, the correct link is `http://users.cms.caltech.edu/~jtropp/papers/HMT11-Finding-Structure-SIREV.pdf`. However, this website doesn't support HTTPS.